icommons_ext_tools/settings/base.py

```diff
@@ -99,23 +99,31 @@
 
 DATABASES = {
     'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': SECURE_SETTINGS.get('db_default_name', 'icommons_ext_tools'),
+        'USER': SECURE_SETTINGS.get('db_default_user', 'postgres'),
+        'PASSWORD': SECURE_SETTINGS.get('db_default_password'),
+        'HOST': SECURE_SETTINGS.get('db_default_host', '127.0.0.1'),
+        'PORT': SECURE_SETTINGS.get('db_default_port', 5432),  # Default postgres port
+    },
+    'coursemanager': {
         'ENGINE': 'django.db.backends.oracle',
-        'NAME': SECURE_SETTINGS.get('django_db', None),
-        'USER': SECURE_SETTINGS.get('django_db_user', None),
-        'PASSWORD': SECURE_SETTINGS.get('django_db_pass', None),
-        'HOST': SECURE_SETTINGS.get('django_db_host', None),
-        'PORT': str(SECURE_SETTINGS.get('django_db_port', None)),
+        'NAME': SECURE_SETTINGS.get('db_coursemanager_name'),
+        'USER': SECURE_SETTINGS.get('db_coursemanager_user'),
+        'PASSWORD': SECURE_SETTINGS.get('db_coursemanager_password'),
+        'HOST': SECURE_SETTINGS.get('db_coursemanager_host'),
+        'PORT': str(SECURE_SETTINGS.get('db_coursemanager_port')),
         'OPTIONS': {
             'threaded': True,
         },
-        'CONN_MAX_AGE': 1200,
+        'CONN_MAX_AGE': 0,
     }
 }
 
 DATABASE_ROUTERS = ['icommons_common.routers.DatabaseAppsRouter']
```

Comment:
Might be easier to go with the `CourseSchemaDatabaseRouter` and avoid the `DATABASE_APPS_MAPPING` setting.